build_compilers/test/summary_builder_test.dart

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:build_test/build_test.dart';
+import 'package:test/test.dart';
+
+import 'package:build_compilers/build_compilers.dart';
+
+import 'matchers.dart';
+
+main() {
+  Map<String, dynamic> assets;
+
+  setUp(() {
+    assets = {
+      'b|lib/b.dart': '''final world = 'world';''',
+      'a|lib/a.dart': '''
+import 'package:b/b.dart';
```

Comment:
Not a fan of the formatting here. Can you manually fix it up?

Reply:
Indented things, usually I don't like to do that because it means the actual file contents have weird indentation but I guess it doesn't matter in this case.